### ISSUE TITLE
Tables not supported for gfm output

### ISSUE THREAD

**Issue by jgm** [kivikakk/cmark-gfm-hs#3]
Line 484 of CMarkGFM.hsc:

```haskell
             TABLE aligns  -> error "constructing table not supported"
             TABLE_ROW     -> error "constructing table row not supported"
             TABLE_CELL    -> error "constructing table cell not supported"
```

It would be great if tables could be supported.  Is there some problem?  Perhaps I can help.

Also:  this library will abort with an uncatchable runtime error, crashing any program that uses it, if you try to render a table node.  That is not good behavior for a library.  `error` should be used only in "should not happen" cases, i.e. for programming errors.

**Comment by jgm** [comment]
Looking into this further, I notice that cmark-gfm lacks any good way to programatically construct the nodes provided by extensions.  So, for example `set_table_alignments` is a static function in tables.c, not an exported function.  Is there a reason for this?

**Comment by juhp** [comment]
Not sure if this is not already fixed?

**Comment by kivikakk** [comment]
Not yet, I don't believe.

**Comment by ingun37** [comment]
Are there any updates?

**Comment by kivikakk** [comment]
No. [`cmark-gfm`](https://github.com/github/cmark-gfm) still lacks the relevant exports mentioned above:

> Looking into this further, I notice that cmark-gfm lacks any good way to programatically construct the nodes provided by extensions. So, for example `set_table_alignments` is a static function in tables.c, not an exported function. Is there a reason for this?

The last commit to `cmark-gfm` was more than 2 years ago; I do not know if it is maintained.
